res/Sectors/M1105/Gzirr!k'l.txt

```diff
@@ -0,0 +1,562 @@
+1831 Nefrark              C353957-C Hi Po                                 - - - 724   Kr M2 V G2 V       
+1832 Husvegg              B89A510-F Ni Wa                                 - - - 602   Kr F1 IV M6 V      
+1833 Emyzi                A866994-F Ga Cp Hi Pr (Krotan)9                 - N - 313   Kr G0 V            
+1835 Qraklojia            A88A545-E Ni Pr Wa                              - N - 623   Kr K9 V            
+1837 Enabbus              E729000-0 Ba                                    - N - 024   Kr M7 V            
+1840 Abwedrint            B88A657-D Ni Ri Wa                              - N - 510   Kr G7 V            
+1931 Grelan               B100757-D Na Pi Va                              - N - 800   Kr K0 V            
+1939 Zogleedaa            A32067B-B He De Na Ni Po                        - - - 315   Kr M9 V M9 V       
+1940 Netwone'ee           B746616-A Ag Ni                                 - - - 805   Kr G8 V            
+2031 Ygrust               A525879-C Pi Ph                                 - N - 823   Kr M5 III M4 V     
+2036 Idwoghia             C66A815-5 Ri Wa Ph                              - - - 802   Kr G0 V M4 V       
+2039 Ruthunt              C362586-A Ni Pr                                 - N - 804   Kr M8 V K5 V       
+2131 Ygrust               A332754-C Na Po                                 - - - 514   Kr M7 V            
+2133 Zikarah Diadem       B000411-A Va As Ni                              - N - 103   Kr M5 V            
+2134 Vruhon               B200230-D Lo Va                                 - N - 822   Kr F5 V            
+2136 Jeflyd               B371540-B He Ni                                 - N - 120   Kr M9 V            
+2137 Vrydvir              E410987-8 Hi In Na                              - - - 702   Kr F2 V            
+2232 Klesast              E788244-8 Lo                                    - - - 803   Kr M5 V M2 V       
+2236 Bwoqrest Diadem      B000302-D Va As Lo                              - - - 603   Kr K3 V M8 V       
+2238 Tsodaggos            A74A101-E Lo Wa                                 - - - 903   Hc A7 V G5 V       
+2332 Obvatsevos           C3617B9-9 Ri                                    - - - 923   Hc M3 V            
+2333 Prilocadd            E614000-0 Ba Ic                                 - - - 010   Hc M1 V            
+2334 Apvekab              B410772-B Na Pi                                 - - - 500   Xh M5 V            
+2335 Vlesland             C478663-6 Ag Ni Cy O:XXXX                       - - - 500   Na G2 V            
+2337 Esvodvujos           E8A0000-0 Ba He                                 - - - 004   Na M4 V            
+2338 Pricogga             E358584-8 Ag Ni                                 - - - 505   Na M5 V            
+2340 Gnufrorte            C453442-7 Ni Po                                 - - - 702   Na M2 III          
+2431 Hezlymus             B040AA9-B As De Hi In Po                        - - - 814   Na M1 V M9 V       
```

Comment:
My parser got confused by this wold. Asteroid worlds are "000", or did you mean for this to be a small word?